### PR TITLE
functions: added invocation metrics

### PR DESCRIPTION
**Pull request by lp-mb**

### CODE REVIEW

**Review by Templum — CHANGES_REQUESTED** [review]

File: pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md [hunk 1]
```diff
@@ -118,3 +118,12 @@ All activities will be stored for roughly two months and can be reviewed in this
 Clicking on an individual activity within the activity stream will bring you to a page based on the content. For example, a `Code changed` activity will bring you to a page comparing the previous and that versions. Things like changes to the allowlisting will get you to the allowlist settings.
 
 <img alt="Functions:Activity Context Action" src="img/functions/functions_reporting_activties_context_action.png"> 
+
+### Invocation Metrics
```
Comment (Templum) [inline pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md:122]:
```suggestion
### Invocation Histogram
```

**Review by Templum — APPROVED** [review]

File: pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md [hunk 2]
```diff
@@ -118,3 +118,12 @@ All activities will be stored for roughly two months and can be reviewed in this
 Clicking on an individual activity within the activity stream will bring you to a page based on the content. For example, a `Code changed` activity will bring you to a page comparing the previous and that versions. Things like changes to the allowlisting will get you to the allowlist settings.
 
 <img alt="Functions:Activity Context Action" src="img/functions/functions_reporting_activties_context_action.png"> 
+
+### Invocation Histogram
```
Comment (j9t) [inline pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md:122]: Please add extra line between heading and paragraph (for consistency with rest of document).

**Review by j9t — CHANGES_REQUESTED** [review]

File: pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md (hunk 2 above)
Comment (j9t) [reply pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md:122]: (Per internal conversation, deferring this to approve and merge now.)

File: pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md [hunk 3]
```diff
@@ -118,3 +118,12 @@ All activities will be stored for roughly two months and can be reviewed in this
...
+The invocation histogram allows you to better understand how your function was invoked over time and how it behaved. The graph includes successful and unsuccessful invocations.
```
Comment (Templum) [inline pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md:123]:
```suggestion

The invocation histogram allows you to better understand how your function was invoked over time and how it behaved. The graph includes successful and unsuccessful invocations.
```

**Review by Templum — COMMENTED** [review]
As requested

**Pull request merged**